From 11d121080a5b7d0b852fc88f758320830b4d1bcd Mon Sep 17 00:00:00 2001
From: Nathan Nutter <iam@nnutter.com>
Date: Thu, 26 Dec 2013 15:09:23 -0600
Subject: patch Makefile for DESTDIR

---
 Makefile | 17 +++--------------
 1 file changed, 3 insertions(+), 14 deletions(-)

diff --git a/Makefile b/Makefile
index a35e994..e254a1f 100644
--- a/Makefile
+++ b/Makefile
@@ -1,25 +1,14 @@
 #!/usr/bin/make -f
 
-prefix ?= /usr/local
+prefix ?= $(DESTDIR)/usr
 exec_prefix ?= $(prefix)
 
 bindir ?= $(exec_prefix)/bin
-sysconfdir ?= $(prefix)/etc
+sysconfdir ?= $(DESTDIR)/etc
 
 profiledir = $(sysconfdir)/profile.d
 
-all: install caveats
-
-caveats:
-	@echo 'If $(profiledir)/*.sh are not already automatically sourced you should add the following to your ~/.bash_profile:'
-	@echo
-	@echo '  for script in $(profiledir)/*.sh; do'
-	@echo '    if [ -r $$script ]; then'
-	@echo '      source $$script'
-	@echo '    fi'
-	@echo '  done'
-	@echo
-	@echo "This will enable the use of the 'bd' command which makes it easy to switch between a repo's branchdirs. 'git-bd' is available regardless of whether you choose to do this."
+all: install
 
 install:
 	@install -d $(bindir)
-- 
1.8.4

