From 9ab998d121546a13d86ab2740bc368f864a5d4ea Mon Sep 17 00:00:00 2001
From: Nathan Nutter <iam@nnutter.com>
Date: Thu, 26 Dec 2013 15:17:34 -0600
Subject: make fake build and clean targets

---
 Makefile | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/Makefile b/Makefile
index b39f220..a8f33bf 100644
--- a/Makefile
+++ b/Makefile
@@ -34,17 +34,6 @@ git-bd.1: git-bd.md
 
 docs: git-bd.1
 
-caveats:
-	@echo 'If $(profiledir)/*.sh are not already automatically sourced you should add the following to your ~/.bash_profile:'
-	@echo
-	@echo '  for script in $(profiledir)/*.sh; do'
-	@echo '    if [ -r $$script ]; then'
-	@echo '      source $$script'
-	@echo '    fi'
-	@echo '  done'
-	@echo
-	@echo "This will enable the use of the 'bd' command which makes it easy to switch between a repo's branchdirs. 'git-bd' is available regardless of whether you choose to do this."
-
 install:
 	@install -d $(DESTDIR)$(bindir)
 	@install git-bd $(DESTDIR)$(bindir)/git-bd
-- 
1.8.4

