From 4950727b8ef94967a801b7f010ae337b7fb78fc1 Mon Sep 17 00:00:00 2001
From: Nathan Nutter <iam@nnutter.com>
Date: Thu, 26 Dec 2013 15:17:34 -0600
Subject: make fake build and clean targets

---
 Makefile | 17 +++++------------
 1 file changed, 5 insertions(+), 12 deletions(-)

diff --git a/Makefile b/Makefile
index 36a6287..bd7034d 100644
--- a/Makefile
+++ b/Makefile
@@ -23,18 +23,11 @@ PROFILE_SH=\
 'source $(bashrc)\n'\
 'fi'
 
-all: install caveats
-
-caveats:
-	@echo 'If $(profiledir)/*.sh are not already automatically sourced you should add the following to your ~/.bash_profile:'
-	@echo
-	@echo '  for script in $(profiledir)/*.sh; do'
-	@echo '    if [ -r $$script ]; then'
-	@echo '      source $$script'
-	@echo '    fi'
-	@echo '  done'
-	@echo
-	@echo "This will enable the use of the 'bd' command which makes it easy to switch between a repo's branchdirs. 'git-bd' is available regardless of whether you choose to do this."
+build:
+	true
+
+clean:
+	true
 
 install:
 	@install -d $(DESTDIR)$(bindir)
-- 
1.8.4

