From a6ceb513afc2a345154cba9fc366ce614d0540f3 Mon Sep 17 00:00:00 2001
From: Nathan Nutter <iam@nnutter.com>
Date: Thu, 26 Dec 2013 15:17:34 -0600
Subject: make fake build and clean targets

---
 Makefile | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e254a1f..6f2cd6f 100644
--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,11 @@ sysconfdir ?= $(DESTDIR)/etc
 
 profiledir = $(sysconfdir)/profile.d
 
-all: install
+build:
+	true
+
+clean:
+	true
 
 install:
 	@install -d $(bindir)
-- 
1.8.4

